<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_computer_dialog</class>
 <widget class="QDialog" name="add_computer_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="name_computer_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_computer_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_computer_type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_computer_year_built">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="type_computer_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="year_built_computer_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year built:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>10</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>Please input information about computer in the fields below</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new_computer_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new computer!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
